src/main/java/com/uipath/uipathpackage/Utility.java

```diff
@@ -128,46 +75,53 @@ private void copyPluginFiles(@Nonnull TaskListener listener, File tempDir, Strin
             while (jellies.hasMoreElements()) {
                 URL jellyU = jellies.nextElement();
                 if (jellyU.getProtocol().equals("file")) {
-                    File jellyF;
-                    try {
-                        jellyF = new File(jellyU.toURI());
-                    } catch (URISyntaxException e) {
-                        e.printStackTrace(listener.getLogger());
-                        throw e;
-                    }
+                    File jellyF = new File(jellyU.toURI());
                     File classes = jellyF.getParentFile();
                     if (classes.getName().equals("classes")) {
-                        execute(listener, "Copy-Item -Path " + escapePowerShellString(classes.getAbsolutePath() + "\\*") + " -Destination " + escapePowerShellString(tempDir.getAbsolutePath()) + " -Recurse -force");
+                        FilePath wsClasses = new FilePath(classes);
+                        wsClasses.copyRecursiveTo(tempDir);
                         listener.getLogger().println("Files copied to temp");
                     }
                 }
             }
         } else {
             listener.getLogger().println("extracting powershell modules to temp folder");
-            extractResourcesToTempFolder(tempDir, jar, listener);
+            extractResourcesToTempFolder(tempDir, jar, listener, rb);
             listener.getLogger().println("extracted powershell modules to temp folder");
         }
     }
 
-    private File getTempDir() throws IOException {
-        File baseDir = new File(System.getProperty("java.io.tmpdir"));
-        File tempDir = new File(baseDir, "UiPath");
-        if (!tempDir.exists()) {
-            boolean result = tempDir.mkdir();
-            if (!result) {
-                throw new AbortException("Failed to create temp directory");
-            }
+    /**
+     * Wraps a string in quotes and escapes all PowerShell special characters.
+     * This is a helper method for creating strings that will be interpreted
+     * literally by PowerShell.
+     *
+     * @param s the string to be escaped.
+     * @return an escaped string.
+     */
+    public String escapePowerShellString(String s) {
+        return "'" + s.replace("'", "''") + "'";
+    }
+
+    public String getValue(ResourceBundle rb, String s) {
```

Comment:
Please consistently follow coding patterns. We need to have documentation on all public members.